MAINTAINING.md

```diff
@@ -0,0 +1,76 @@
+# RIOT Maintainer Guidelines
+
+This list presents a series of guidelines for maintainers, including technical and non-technical. The list is not exhaustive, and represents a baseline on things that should be ensured for contributions.
+
+Notes:
+- Any of the items in this list might be skipped, if clearly and logically articulated reasons are presented.
+- The order of the steps is meant to maximize efficiency and minimize overhead, with the philosophy that failing in step n makes steps n+x obsolete. However, this efficiency can depend on the quality of the submission itself. If the PR is clearly not in a reviewable state, for example due to poor code cleanliness or overall design, it might be more efficient to give the contributor some broader comments for improvement before further review.
```

Comment:
This is the first occurrence of the abbreviation "PR". Though maintainers most likely should know what a PR is, I think it's more easier to follow, if PR is resolved here at least: "PR (Pull Request)".

Reply:
done